pyproject.toml

```diff
@@ -51,7 +53,10 @@ cryptography = "42.0.3"
 aws-logging-handlers = "^2.0.5"
 s3fs = "^2024.3.1"
 fsspec = "^2024.3.1"
-villas-python = {git = "https://github.com/VILLASframework/node.git", rev = "b66b125ea192faa876fc18ab64fcc798dcdb17ce", subdirectory = "python"}
+villas-python = { url = "https://github.com/VILLASframework/node/archive/bcb83e2733ed7b042fd87f9cfdcde37b1b14226d.tar.gz" }
```

Comment:
Is there a reason you moved to a tarball here? This seems to lead to the current [CI error](https://github.com/SEGuRo-Projekt/Platform/actions/runs/8527105899/job/23357782797?pr=76#step:5:1309), as the VILLASnode archive seems not to be packaged correctly (see: https://github.com/python-poetry/poetry/issues/5793#issuecomment-1149798927). 

I have similar problems on my own system, when updating the poetry lock file:
```
Unable to determine package info from path: <my_home>/.cache/pypoetry/artifacts/13/ac/71/b01cf1c6513ba833a1eb179df5a1a6108fcd60ba629b0104f3c0513935/bcb83e2733ed7b042fd87f9cfdcde37b1b14226d.tar.gz
```

Reply:
Yeah, somehow Poetry + Nix is having issues with the `villas-python` dependency no matter how I specify it.. I think I will package it as a regular Python package and publish it via pypi.org..